README.md

```diff
@@ -10,31 +11,75 @@ This chart will deploy the following on the target OpenShift cluster:
-- `ArgoCD/OpenShift GitOps` operator is installed and one instance of `ArgoCD` exists in a given namespace (later referenced by `ARGOCD_NAMESPACE` env var)
+  - A [reference implementation](https://github.com/parodos-dev/orchestrator-helm-chart#deploying-postgresql-reference-implementation) is provided for your convenience.
+- A GitHub API Token - to import items into the catalog, ensure you have a GITHUB_TOKEN with the necessary permissions as detailed [here](https://backstage.io/docs/integrations/github/locations/). For classic token, include the following permissions:
+  - repo (all)
+  - admin:org (read:org)
+  - user (read:user, user:email)
+
+## Installation
+
+### Quick installation
+1. Follow instruction [here](https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/GitOps.md)
+2. Run the following command to set up environment variables:
+    ```console
+    ./hack/setenv.sh
+    ```
+  - Accept all default values
+  - Provide GH Token (created as describe in prerequisite section)
+3. Source the environment variables by running
+    ```console
+    source .env
+    ```
+4. Install the orchestrator chart using one of the following options:
+   1. **Option 1: Install the chart with SonataFlow services in ephemeral mode for evaluation purposes**
+      ```console
+      helm install orchestrator orchestrator --set orchestrator.devmode=true \
+          --set rhdhOperator.github.token=$GITHUB_TOKEN \
+          --set rhdhOperator.k8s.clusterToken=$K8S_CLUSTER_TOKEN --set rhdhOperator.k8s.clusterUrl=$K8S_CLUSTER_URL \
+          --set argocd.namespace=$ARGOCD_NAMESPACE --set argocd.url=$ARGOCD_URL --set argocd.username=$ARGOCD_USERNAME \
+          --set argocd.password=$ARGOCD_PASSWORD --set argocd.enabled=true --set tekton.enabled=true
+      ```
+   2. **Option 2: Deploy PostgreSQL reference implementation**
+      1. Deploy PostgreSQL reference implementation following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/postgresql/README.md)
+      2. Install the orchestrator Helm chart:
+        ```console
+        helm install orchestrator orchestrator --set rhdhOperator.github.token=$GITHUB_TOKEN \
+          --set rhdhOperator.k8s.clusterToken=$K8S_CLUSTER_TOKEN --set rhdhOperator.k8s.clusterUrl=$K8S_CLUSTER_URL \
+          --set argocd.namespace=$ARGOCD_NAMESPACE --set argocd.url=$ARGOCD_URL --set argocd.username=$ARGOCD_USERNAME \
+          --set argocd.password=$ARGOCD_PASSWORD --set argocd.enabled=true --set tekton.enabled=true
+        ```
+  5. Run the commands prompted at the end of the previous step to wait until the services are ready.
+
+## Detailed Installation
+
+### Additional Prerequisites
+In addition to the [prerequisites](https://github.com/parodos-dev/orchestrator-helm-chart#prerequisites) mentioned earlier, is it possible to manually install the following operator:
```

Comment:
It's possible or is that mandatory? If the operators are not installed, will the orchestrator chart install them and configure them for me?
If that is related to the `-set argocd.enabled=true` and `--set tekton.enabled=true` then mention that instead

Reply:
if you use the helm chart, it will install the operators for you.
In this section I'd like to say that is also possible to install them manually without relying on the chart.

Reply:
Maybe it's worth to clarify it in the README. Other users can have the same doubt. I'll ask for a double check.